src/rashdf/utils.py

```diff
@@ -221,3 +221,19 @@ def get_first_hdf_group(parent_group: h5py.Group) -> Optional[h5py.Group]:
         if isinstance(item, h5py.Group):
             return item
     return None
+
+
+def convert_ras_datetime(dt: datetime) -> str:
```

Comment:
Is this getting added because the `datetime` objects can't be serialized to JSON? I'd argue that's outside the scope of this library, since our goal is to read RAS HDF files and deliver `GeoDataFrame` objects. For the purposes of the unit test, I think the test itself should do what it needs to do to convert datetimes / Timestamps to a serializeable format.

Reply:
removed this and used `pd.Timestamp.isoformat`